cluster/store.go

```diff
@@ -0,0 +1,24 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"github.com/coreos/etcd/clientv3"
```

Comment:
idk if it matters if we're consistent on `github.com/coreos/etcd` and `go.etcd.io/etcd` or not